data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
-Micro Missile Cluster,diableavionics_microcluster2,3,,7500,1500,,100,,5,0,,30,0.5,6,HIGH_EXPLOSIVE,50,,0.2,4,3,0.2,6,6,5,6,,300,5,15,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Array,diableavionics_microarray,3,,17500,1500,,100,,5,0,,400,1.5,40,HIGH_EXPLOSIVE,50,,0.1,4,20,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
-Micro Missile Array,diableavionics_microarrayM,3,,17500,1500,,100,,5,0,,200,1.5,20,HIGH_EXPLOSIVE,50,,0.1,4,10,0.1,6,6,5,6,,300,150,12,25,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Launcher,diableavionics_micromissile,1,,500,3000,,50,,5,6,3,25,1,4,HIGH_EXPLOSIVE,,,0.1,3.9,5,0,6,6,5,6,,250,180,18,50,,,"DO_NOT_AIM, DO_NOT_CONSERVE,","utility10, missile1, rare_bp, diable_weapon_bp",,Diable Avionics,,Saturation,Medium,Poor,,,,,,,,11.24
+Micro Missile Launcher,diableavionics_microbuiltin,1,,500,3000,,50,,5,6,3,25,1,4,HIGH_EXPLOSIVE,,,0.1,3.9,5,0,6,6,5,6,,250,180,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM","utility10, missile1",,Diable Avionics,,Saturation,Medium,Poor,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster_left,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Cluster,diableavionics_microcluster2,3,,7500,3000,,50,,5,0,,90,2.5,10,HIGH_EXPLOSIVE,,,0.2,3.8,15,0.1,6,6,5,6,,250,5,21,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarray,3,,17500,3000,,50,,5,0,,960,8,32,HIGH_EXPLOSIVE,,,0.1,3.9,80,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
+Micro Missile Array,diableavionics_microarrayM,3,,17500,3000,,50,,5,0,,480,4,16,HIGH_EXPLOSIVE,,,0.1,3.9,40,0.1,6,6,5,6,,250,150,18,50,,,"SYSTEM, DO_NOT_AIM, DO_NOT_CONSERVE, DO_NOT_AIM",,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Magicbox PD Missile,diableavionics_magicbox,1,0.75,400,1000,,150,,3,15,5,8,0.15,1,FRAGMENTATION,25,,0.5,1,1,0,7,14,14,12,,475,35,3.7,40,,,"DO_NOT_AIM, DO_NOT_CONSERVE, PD_ONLY, PD","pd9, missile0, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),Excelent,Excelent,,,,,,,,11.23
+Magicbox PD Missile,diableavionics_magicbox,1,0.75,400,1200,,150,,3,15,5,12,0.5,3,FRAGMENTATION,25,,0.5,1,1,0,7,14,14,12,,475,35,3.7,40,,,"DO_NOT_AIM, DO_NOT_CONSERVE, PD_ONLY, ","pd9, missile0, diable_weapon_bp, rare_bp",,Diable Avionics,,Point Defense (Area),Excelent,Excelent,,,,,,,,11.23
```

Comment:
Magicbox PD Missile range increased to 1200 (from 1000), max ammo increased to 12 (from 8), ammo regen increased to 0.5 per second (from 0.15 per second), ammo reload size set to 3 (from 1)